### PR TITLE
0.7.1

### PR DESCRIPTION
**Pull request by charlierudolph**
@allewun @kevgo @ricmatsui

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,5 +1,19 @@
 # Git Town Release Notes
 
+## 0.7.1 (2015-08-27)
```
Comment (allewun) [inline RELEASE_NOTES.md]: nicely anticipated

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -1,5 +1,19 @@
...
+  ([#593](https://github.com/Originate/git-town/issues/593))
+* remove empty line printed in new-pull-request/repo
+  ([#602](https://github.com/Originate/git-town/issues/602))
+* git kill: prompt for unknown parent branch
+  ([#603](https://github.com/Originate/git-town/issues/603))
+* git sync --all: prompt for unknown parent branch
+  ([#604](https://github.com/Originate/git-town/issues/604))
+* support branch names with forward slashes (along with any valid branch name)
+  ([#608](https://github.com/Originate/git-town/issues/608))
+* fix casing of `git town config` output
```
Comment (allewun) [inline RELEASE_NOTES.md]: seems too insignificant to mention IMO

**Comment by allewun** [comment]
Good from my end

**Comment by charlierudolph** [comment]
Manually merged in 0cc716304800848b4241212c0aef7471d1308e81 as I couldn't ship it :disappointed:

**Comment by kevgo** [comment]
Wow, a lot of changes in a very short time! Nice!

**Pull request closed without merging**